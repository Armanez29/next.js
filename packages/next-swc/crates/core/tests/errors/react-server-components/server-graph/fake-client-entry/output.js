export default function() {
    return null;
}
export const runtime = 'edge';
