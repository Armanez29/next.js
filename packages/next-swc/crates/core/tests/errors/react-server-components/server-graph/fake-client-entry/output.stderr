
  x NEXT_RSC_ERR_CLIENT_DIRECTIVE
   ,-[input.js:4:1]
 4 | 
 5 | 'use client'
   : ^^^^^^^^^^^^
   `----
